server-base/src/main/java/org/apache/kylin/rest/service/CubeService.java

```diff
@@ -573,6 +573,18 @@ public void updateCubeNotifyList(CubeInstance cube, List<String> notifyList) thr
         getCubeDescManager().updateCubeDesc(desc);
     }
 
+    public CubeInstance updateCubeOwner(CubeInstance cube, String owner) throws IOException {
+        aclEvaluate.checkProjectWritePermission(cube);
+        if (cube.getOwner().equals(owner)) {
```

Comment:
How about using Objects.equals() in case of that cube.getOwner() is null?

Reply:
got that. Will update PR later